base/include/FacialLandmarksCV.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include "Module.h"
+
+class Detail;
+class DetailSSD;
+class DetailHCASCADE;
+
+class FacialLandmarkCVProps : public ModuleProps
```

Comment:
1. add serialization 
2. add get set props 
3. add get set props

Reply:
added(no we can change the algorithm type using setProps)